test/testsuites/packagestests.go

```diff
@@ -13,10 +13,17 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// TestCscWithNonExistingPackage tests that a csc with a non-existing package
+// InvalidPackages is a test suite that ensures that package related issues are handled
```

Comment:
You should say `invalid packages` if this test suite only deals with invalid package scenarios.

Reply:
I will rename the test suite to `Packages`.